man/systemd-analyze.xml

```diff
@@ -243,10 +244,12 @@ multi-user.target @47.820s
     </refsect2>
 
     <refsect2>
-      <title><command>systemd-analyze dump</command></title>
+      <title><command>systemd-analyze dump [<replaceable>pattern</replaceable>…]</command></title>
 
-      <para>This command outputs a (usually very long) human-readable serialization of the complete server
-      state. Its format is subject to change without notice and should not be parsed by applications.</para>
+      <para>Without any parameter, this command outputs a (usually very long) human-readable serialization of
+      the complete server state. Optional glob pattern may be specified, causing the output to be limited to
```

Comment:
Not your fault, but maybe `server state` -> `service state`??

Reply:
you mean "service manager state", don't you ?